usr/local/bin/apt

```diff
@@ -13,6 +13,7 @@ def usage():
     print("Commands:")
     print("  add-repository   - Add entries to apt sources.list")
     print("  autoclean        - Erase old downloaded archive files")
+    print("  autopurge        - Purge packages with automatically all unused packages")
```

Comment:
The description doesn't make sense - perhaps it should be `Purge all unused packages automatically`